### DISCUSSION TITLE
Airflow Dynamically generated DAG never gets scheduled (next DagRun is None)

### DISCUSSION THREAD

**Discussion by colax94** [apache/airflow#40893] [Q&A]
I have a dynamically generated DAG (with SQLMesh), which is correctly parsed by Airflow, and the DAG shows up in the UI. The problem is that it never gets scheduled, and the "Next DagRun date" is None. 
The schedule is valid, I tried `@daily` expressions as well as regular cron expressions (`0 * * * *`), but the behaviour is still the same, the DAG never gets scheduled, and the next DagRun date stays Null.

Here is a screenshot of the DAG details : 
<img width="1495" alt="Screenshot 2024-07-19 at 17 23 57" src="https://github.com/user-attachments/assets/3d51c029-d99d-4fe5-9249-b3540536b0bb">

PS : the DAG runs normally when I manually trigger it, but then it is never automatically rescheduled.

**Comment by dondaum** [comment]
What is the Dag's start date?

**Comment by colax94** [comment]
Thanks for looking into this, it looks like I had an incompatible versionsof pendulum package in my Airflow setup. I released the constraint on pendulum and now everything works fine.
